pandas/_libs/tslibs/timedeltas.pxd

```diff
@@ -6,3 +6,6 @@ from numpy cimport int64_t
 cdef int64_t cast_from_unit(object ts, object unit) except? -1
 cpdef int64_t delta_to_nanoseconds(delta) except? -1
 cpdef convert_to_timedelta64(object ts, object unit)
+
+# Exposed for Timedelta, not intended for outside use.
```

Comment:
Is this cimported by another cython module?  If not, doesn’t need to be in the pxd file